code/modules/food&drinks/drinks/drinks.dm

```diff
@@ -288,3 +298,302 @@
 	desc = "A delicious mixture of 42 different flavors."
 	icon_state = "dr_gibb"
 	list_reagents = list("dr_gibb" = 30)
+
+/obj/item/weapon/reagent_containers/food/drinks/soda_cans/pubber
+	name = "Dr. Pubber"
+	desc = "The beverage of an original crowd. Tastes like an industrial tranquilizer."
+	icon_state = "dr_gibb"
+	New()
+		..()
+		reagents.add_reagent("haloperidol", 4)
```

Comment:
Yeah, seeing as we don't use the term "pubbies", don't WANT people using the term "pubbies", and don't have any distinction between "pubbies" and "not pubbies", this should go.